### PR TITLE
[NetAppFiles] Remove deprected properties from backups

### PR DESCRIPTION
**Pull request by audunn**
## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [ ] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by vidai-msft** [comment]
@audunn Please update the change log.

**Comment by vidai-msft** [comment]
@audunn Some parameters for the affected cmdlets are removed. Please update the examples in the help docs as per detected in the Static Analyzer. The previous cmdlet not valid issue has been resolved.

**Comment by audunn** [comment]
can we update the `Update-MarkdownHelpModule` so the process actions thing is not added only to be removed ?

File: src/NetAppFiles/NetAppFiles/ChangeLog.md [hunk 1]
```diff
@@ -18,7 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
+Breaking change:
```
Comment (vidai-msft) [inline src/NetAppFiles/NetAppFiles/ChangeLog.md:21]: Better remove this line as it does not start with a star (*). I am not sure if that will break the version bump-up script.

**Pull request merged**